packages/msi/msi/Product.wxs

```diff
@@ -21,14 +31,15 @@
 <?else?>
 <?error MondooSKU must be defined as one of "standard" or "enterprise" ?>
 <?endif?>
+<?endif?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"
   xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
   <Product Name="$(var.ProductName)" Version="$(var.ProductVersion)" Manufacturer="Mondoo, Inc." Language="1033" Codepage="1252" Id="*" UpgradeCode="$(var.UpgradeCode)">
     <!-- custom action do not work if its not privileged-->
     <Package
       Description="Mondoo verifies your system for known vulnerabilities"
       Manufacturer="Mondoo, Inc."
-      InstallerVersion="200"
+      InstallerVersion="$(var.InstallerVersion)"
       Compressed="yes"
       Comments="Windows Installer Package"
       Platform="x64"
```

Comment:
shouldn't there be an `arm` section too?

Reply:
removed the Platform attribute entirely - it's considered deprecated and is overriden by passing `-arch` to `candle.exe` anyway